### ISSUE TITLE
Gradients do not exist for variables

### ISSUE THREAD

**Issue by CanshangD** [dragen1860/TensorFlow2.0Tutorials#39]
I have got the error “Gradients do not exist for variables” when training faster rcnn， and the loss is nan. 

**log：**
W1108 09:59:39.735346 139647501481728 optimizer_v2.py:1029] Gradients do not exist for variables ['faster_rcnn/b_box_head/rcnn_class_conv1/kernel:0', 'faster_rcnn/b_box_head/rcnn_class_conv1/bias:0', 'faster_rcnn/b_box_head/rcnn_class_bn1/gamma:0', 'faster_rcnn/b_box_head/rcnn_class_bn1/beta:0', 'faster_rcnn/b_box_head/rcnn_class_conv2/kernel:0', 'faster_rcnn/b_box_head/rcnn_class_conv2/bias:0', 'faster_rcnn/b_box_head/rcnn_class_bn2/gamma:0', 'faster_rcnn/b_box_head/rcnn_class_bn2/beta:0', 'faster_rcnn/b_box_head/rcnn_class_logits/kernel:0', 'faster_rcnn/b_box_head/rcnn_class_logits/bias:0', 'faster_rcnn/b_box_head/rcnn_bbox_fc/kernel:0', 'faster_rcnn/b_box_head/rcnn_bbox_fc/bias:0'] when minimizing the loss.
epoch 0 2 nan

**Comment by MLForNerds** [comment]
Got the same issue, couldn't solve it. Kindly post the solution if you have solved it.

**Comment by IngLP** [comment]
I have the same issue and for me too the loss is NaN.
Please help

**Comment by xiaolng** [comment]
same issue, loss becomes nan

**Comment by flori-shan** [comment]
same issue, please help

**Comment by jgarza9788** [comment]
can you please say what file you were running when you got this error?
what version(s) of pytorch and python etc ?
